src/wui/interactive_base.cc

```diff
@@ -770,6 +770,30 @@ void InteractiveBase::blit_field_overlay(RenderTarget* dst,
 	blit_overlay(dst, field.rendertarget_pixel.cast<int>(), image, hotspot, scale);
 }
 
+void InteractiveBase::draw_bridges(RenderTarget* dst,
+                                   const FieldsToDraw::Field* f,
+                                   uint32_t gametime,
+                                   float scale) const {
+	if (Widelands::is_bridge_segment(static_cast<Widelands::RoadSegment>(f->road_e))) {
```

Comment:
Is there a reason that `f->road_e`  etc. are `uint8_t` instead of `Widelands::RoadSegment`?

Reply:
A leftover from when this was in `logic` code. I´m making them RoadSegment now.